packages/web3-core-method/src/methods/transaction/GetPendingTransactionMethod.js

```diff
@@ -0,0 +1,36 @@
+/*
+    This file is part of web3.js.
+
+    web3.js is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    web3.js is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @file GetPendingTransactionMethod.js
+ * @author Prince Sinha <sinhaprince013@gmail.com>
+ * @date 2018
+ */
+
+import AbstractMethod from '../../../lib/methods/AbstractMethod';
+
+export default class GetPendingTransactionMethod extends AbstractMethod {
+    /**
+     * @param {Utils} utils
+     * @param {Object} formatters
+     * @param {AbstractWeb3Module} moduleInstance
+     *
+     * @constructor
+     */
+    constructor(utils, formatters, moduleInstance) {
+        super('eth_pendingTransactions', 0, utils, formatters, moduleInstance);
```

Comment:
Where did you see this JSON-RPC method documented? I could only find the [parity](https://wiki.parity.io/JSONRPC-parity-module#parity_pendingtransactions) version of it. @princesinha19

Reply:
@nivida There is no any documentation available for eth_pendingTransactions. But, I tested on Geth and Ganache and it was working.

Reply:
@nivida It seems that this method is not supported by Ganache. So, I created an issue in their repo.
https://github.com/trufflesuite/ganache-cli/issues/638

Reply:
I also updated the JSON_RPC wiki.
https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_pendingtransactions